pallets/foreign-asset-creator/src/tests.rs

```diff
@@ -41,8 +41,15 @@ fn creating_foreign_works() {
 		);
 		expect_events(vec![crate::Event::ForeignAssetCreated {
 			asset_id: 1,
-			foreign_asset: Location::parent(),
-		}])
+			foreign_asset: MultiLocation::parent(),
```

Comment:
The `Multi` prefix is deprecated now.

```suggestion
			foreign_asset: Location::parent(),
```